doc/how-to/add_service.rst

```diff
@@ -0,0 +1,41 @@
+.. _howto-add-service:
+
+How to add a new service
+========================
+
+If you set up the MicroCloud without MicroOVN or MicroCeph initially, you can add those services with the command :command:`microcloud service add`::
+
+  sudo microcloud service add
+
+If MicroCloud detects a service is installed but not set up, it will ask to configure the service.
+
+#. Select whether you want to set up distributed storage (if adding MicroCeph to the MicroCloud).
```

Comment:
This took me a bit to understand. I think it would be clearer to split the instructions into two parts, one for MicroCeph and one for MicroOVN. The drawback is that we don't have one full list of steps for running the tool, but I think it will still be easy to follow. And users who want to add only one service can skip the part for the other one without wondering if they are missing anything.

```suggestion
To add MicroCeph:

1. Select ``yes`` to set up distributed storage.
```